src/DIRAC/TransformationSystem/Utilities/ScriptUtilities.py

```diff
@@ -0,0 +1,59 @@
+"""Utilities used by TS scripts."""
+
+from DIRAC import gLogger
+from DIRAC.TransformationSystem.Client.TransformationClient import TransformationClient
+
+
+def _getTransformationID(transName):
+    """Check that a transformation exists and return its ID or None if it doesn't
+    exist.
+
+    :param transName: name or ID of a transformation
+    :type transName: int or string
+
+    :return : transformation ID or None if it doesn't exist
+    """
+    testName = transName
+    trClient = TransformationClient()
+    # We can try out a long range of indices, as when the transformation is not found, it returns None
+    for ind in range(1, 100):
+        result = trClient.getTransformation(testName)
+        if not result["OK"]:
+            # Transformation doesn't exist
+            break
+        status = result["Value"]["Status"]
+        # If the status is still compatible, accept
+        if status in ("Active", "Idle", "New", "Stopped", "Completed"):
+            return result["Value"]["TransformationID"]
+        # If transformationID was given, return error
+        if isinstance(transName, int) or transName.isdigit():
+            gLogger.error("Transformation in incorrect status", f"{testName}, status {status}")
+            return False
+        # Transformation name given, try out adding an index
+        testName = f"{transName}-{ind}"
+    return None
+
+
+def getTransformations(args):
+    """Parse the arguments of the script and generates a list of transformations."""
+    transList = []
+    if len(args):
```

Comment:
```suggestion
    if not args:
        return []
```
and save on the indentation